stubs/console-menu/consolemenu/screen.pyi

```diff
@@ -0,0 +1,17 @@
+from typing import Any
+
+class Screen:
+    def __init__(self) -> None: ...
+    @property
+    def screen_height(self) -> int: ...
+    @property
+    def screen_width(self) -> int: ...
+    @staticmethod
+    def clear() -> None: ...
+    @staticmethod
+    def flush() -> None: ...
+    def input(self, prompt: str = ...) -> str: ...
+    @staticmethod
+    def printf(*args: list[Any]) -> None: ...
+    @staticmethod
+    def println(*args: list[Any]) -> None: ...
```

Comment:
```suggestion
    def printf(*args: Any -> None: ...
    @staticmethod
    def println(*args: Any) -> None: ...
```

Also, I suspect we can change these to `object` since the args are just passed to `print()`.